packages/codegen-openapi/README.md

```diff
@@ -0,0 +1,55 @@
+# 🏛️ Hyper Fetch Openapi Request Generator 
+
+<p>
+  <a href="https://bettertyped.com/">
+    <img src="https://custom-icon-badges.demolab.com/static/v1?label=&message=BetterTyped&color=333&logo=BT" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/github/stars/BetterTyped/hyper-fetch?logo=star&color=118ab2" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch/blob/main/License.md">
+    <img src="https://custom-icon-badges.demolab.com/github/license/BetterTyped/hyper-fetch?logo=law&color=yellow" />
+  </a>
+  <a href="https://www.npmjs.com/package/@hyper-fetch/graphql">
+    <img src="https://custom-icon-badges.demolab.com/npm/v/@hyper-fetch/graphql.svg?logo=npm&color=e76f51" />
+  </a>
+  <a href="https://api.codeclimate.com/v1/badges/eade9435e75ecea0c004/test_coverage">
+    <img src="https://api.codeclimate.com/v1/badges/eade9435e75ecea0c004/test_coverage" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/badge/typescript-%23007ACC.svg?logo=typescript&logoColor=white" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/badge/-Axios-E10098?logo=axios&logoColor=white" />
+  </a>
+  <a href="https://www.npmjs.com/package/@hyper-fetch/axios">
+    <img src="https://custom-icon-badges.demolab.com/bundlephobia/minzip/@hyper-fetch/axios?color=64BC4B&logo=package" />
+  </a>
+</p>
+
+## About
+
+**`Hyper Fetch Generator-Openapi`** is our request generator from openapi (V3) json schema. Just provide a file path or link and generate all the requests. 
+
+## Features
+
+- Generate code from OpenApi V3 schema - both types and actual requests!
+- Flexible - after generation you can edit Hyper Fetch client and all related requests!
+- No additional boilerplate - if you know Hyper Fetch, you don't need anything else.
+
+## Help us keep working on this project ❤️
+
+- [Become a Sponsor on GitHub](https://github.com/sponsors/prc5)
+
```

Comment:
Can we add the simple examples here and in adapters of usage? It could be just like: 


```tsx
npx @hyper-fetch/codegen-openapi --schema https://petstore3.swagger.io/api/v3/openapi.json
```